### PR TITLE
Added proper ssh with shell escaping

### PR DESCRIPTION
**Pull request by abhishekkushwaha4u**

### CODE REVIEW

File: src/ssh_via_subprocess.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import subprocess
+import json
+import asyncio
+from shlex import join, quote
+
+
+async def run_remote_commands_for_data(username, password, host, url_to_curl, port='22'):
+    command = "sshpass -p {} ssh -p {} -o StrictHostKeyChecking=no {}@{} 'curl -s {}'".format(quote(password), quote(str(port)), quote(username), quote(host), quote(url_to_curl))
```
Comment (Geek-ubaid) [inline src/ssh_via_subprocess.py:8]: @abhishekkushwaha4u you are still using the same formatting way to create the subprocess command?

File: src/ssh_via_subprocess.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        print(host," ",e, " ", error)
+        return {"error": "We have some error in running some commands into your server, please check your creds once, or try checking netdata installation!"}
+    else:
+        print("This is error while executing!")
+        print(host, " " ,error)
+        return {"error": "Unable to execute this operation!"}
+
+async def check_valid_ssh_and_netdata(username, password, host, port='22'):
+    url_to_curl = 'http://localhost:19999/api/v1/info'
+    command = "sshpass -p {} ssh -p {} -o StrictHostKeyChecking=no {}@{} 'curl -s {}'".format(quote(password), quote(str(port)), quote(username), quote(host), quote(url_to_curl))
```
Comment (Geek-ubaid) [inline src/ssh_via_subprocess.py:33]: Same formatting command here also

**Review by Geek-ubaid — CHANGES_REQUESTED** [review]

File: src/ssh_via_subprocess.py (hunk 2 above)
Comment (abhishekkushwaha4u) [reply src/ssh_via_subprocess.py:33]: have used the quote function for shell escaping.
Imported the same, according to docs.
https://docs.python.org/3/library/shlex.html

File: src/ssh_via_subprocess.py (hunk 1 above)
Comment (abhishekkushwaha4u) [reply src/ssh_via_subprocess.py:8]: I did it with official docs, don't worry, this is safe now.

**Pull request merged**